components/esp_modem/examples/modem_tcp_client/main/sock_commands.cpp

```diff
@@ -0,0 +1,185 @@
+/*
+ * SPDX-FileCopyrightText: 2022-2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <charconv>
+#include <cstring>
+#include "sock_commands.hpp"
+
+namespace sock_commands {
+
+static const char *TAG = "sock_commands";
+
+using namespace esp_modem;
+
+command_result net_open(CommandableIf *t)
```

Comment:
Could `t` have a better name?

Reply:
Have renamed to `term` (this could be any class, that implements *commands* and as such, we call it *"commandable"*, but there are in most cases just more or less abstracted *terminals*